pandas/tests/io/test_parquet.py

```diff
@@ -672,6 +675,16 @@ def test_read_empty_array(self, pa, dtype):
             df, pa, read_kwargs={"dtype_backend": "numpy_nullable"}, expected=expected
         )
 
+    @pytest.mark.network
+    @pytest.mark.single_cpu
+    def test_parquet_read_from_url(self, httpserver, datapath, df_compat, engine):
```

Comment:
This test is just moved down from the `Base` class to `TestBasic` class (all tests that are generic and run for both engines are in this second class), but fixing that the `engine` is actually used in the read_parquet call